ledger-service/http-json/BUILD.bazel

```diff
@@ -649,6 +797,35 @@ test_suite(
     ]
 ]
 
+alias(
+    name = "failure-tests-lib",
+    actual = "failure-tests-pkg-ce",
+    visibility = ["//test-evidence:__pkg__"],
+)
+
+[
+    da_scala_test_suite(
+        name = "failure-tests-{}".format(edition),
+        data = [
+            ":Account.dar",
+            "//docs:quickstart-model.dar",
+            "//ledger/test-common:dar-files",
+            "//ledger/test-common/test-certificates",
+            "@toxiproxy_dev_env//:bin/toxiproxy-cmd" if not is_windows else "@toxiproxy_dev_env//:toxiproxy-server-windows-amd64.exe",
+        ],
+        # See https://github.com/digital-asset/daml/issues/9886
+        flaky = True,
+        resources = glob(["src/failure/resources/**/*"]),
+        deps = [
+            "failure-tests-pkg-{}".format(edition),
```

Comment:
Same question here, as this is a completely different strategy from what you're doing with the integration tests.